src/renderer/containers/Onboard/First/Entrance.tsx

```diff
@@ -0,0 +1,19 @@
+import React, {ReactElement} from "react";
+import {Link} from "react-router-dom";
+import {ButtonSecondary, ButtonPrimary} from "../../../components/Button/ButtonStandard";
+import {Routes, Subroutes} from "../../../constants/routes";
+
+export default function Entrance(): ReactElement {
```

Comment:
SigningKey (in directory with same name) and than you can have SigningKeyImport and SigningKeyGenerate?